packages/components/src/components/toggle-switch/toggle-switch.scss

```diff
@@ -6,35 +6,27 @@
     &__label {
         display: flex;
         align-items: center;
-        justify-content: space-between;
+        margin-left: auto;
+        margin-right: 1.6rem;
+        padding-left: 0.4rem;
         cursor: pointer;
-        width: 100px;
-        height: 50px;
+        width: 4.9rem;
+        height: 2.55rem;
         background: var(--general-disabled);
-        border-radius: 100px;
-        position: relative;
+        border-radius: 4.9rem;
         transition: background-color 0.25s;
-
-        &:active .dc-toggle-switch__button {
-            width: 55%;
-        }
     }
     &__button {
-        content: '';
-        position: absolute;
-        top: 6px;
-        left: 8px;
-        width: 37%;
-        height: 75%;
-        border-radius: 50%;
-        transition: 0.25s;
+        width: 1.9rem;
+        height: 1.9rem;
+        border-radius: 1.9rem;
+        transition: transform 0.25s;
         background: var(--text-colored-background);
     }
     &:checked + &__label {
         background: var(--text-profit-success);
     }
     &:checked + &__label &__button {
-        left: calc(100% - 8px);
-        transform: translateX(-100%);
+        transform: translateX(2.2rem);
```

Comment:
Will the translation look similar in the responsive view?

Reply:
I hope so, I checked it with different responsive width. both variants worked well (% and exact value).
I decided to stay with exact value as we have exact value of toggle width.